docs/Automation.md

```diff
@@ -1,183 +1,196 @@
+config | `Config` | The configuration used by the returned `IScanner` object.
 
-#### Stop Command
-The Stop command terminates an automation session. State and associated
-resources are freed. The test framework can then exit or create another session
-(via the Start command).
+##### Return object
 
-##### Inputs
+The **ScannerFactory.CreateScanner** method returns an **IScanner** object.
 
-###### .NET
+#### IScanner
 
-The **StopCommand.Execute** method accepts no parameters.
+##### Scan
+The Scan runs AxeWindows automated tests using the config provided at the time of creation of the scanner.
 
-###### PowerShell
+##### Parameters
 
-The **Stop-AxeWindows** cmdlet accepts no parameters
+The **Scan** method accepts no parameters.
 
 ##### Return object
-The StopCommandResult object has the following properties:
+
+The **Scan** method returns a **ScanResults** object and has the following properties:
 
 **Name** | **Type** | **Description**
 ---|---|---
-Completed | `bool` | True if the command ran to completion. If true, then the Succeeded property indicates if the call succeeded.
-SummaryMessage | `string` | A human-readable message to summarize the result. This message can change at any time, so do not rely on parsing it in tools.
-Succeeded | `bool` | True if the Start command completed successfully.
+ErrorCount | `int` | A count of all errors across all elements scanned.
+Errors | `IEnumerable<ScanResult>` | A collection of errors found during the scan.
+OutputFile | `(string A11yTest, string Sarif)` | A Tuple with paths to any output files written as a result of a scan. Tuple members are A11yTest and Sarif (**not implemented yet**).
 
-#### Defined Constants
+The **Errors** property contains **ScanResult** objects which are the result of a single rule test on a single element and have the following properties:
 
-The following constants are defined in the assembly:
+**Name** | **Type** | **Description**
+---|---|---
+Rule | `RuleInfo` | Information about the rule (description, how to fix information, etc.) that was evaluated on the element.
+Element | `ElementInfo` | The element which was tested against the rule.
 
-**Name** | **Value** | **Description**
+`RuleInfo` contains the following properties:
+
+**Name** | **Type** | **Description**
+---|---|---
+ID | `RuleId` | Contains a unique identifier for the rule from the RuleId enumeration.
+Description | `string` | Contains a short description of the rule.
+HowToFix | `string` | Detailed information on how to resolve a violation reported by the rule.
+Standard | `A11yCriteriaId` | An enum which identifies the standards documentation from which the rule was derived.
+PropertyID | `int` | In cases where the rule tests one specific UI Automation property, this contains the UI Automation property ID in question. This property is used to link elements with rule violations to relevant documentation.
+Condition | 'string' | A description of the conditions under which a rule will be evaluated.
```

Comment:
`string` here is using single quotes instead of backticks so the text does not look monotyped